src/System.Memory/src/System/Buffers/IBufferWriter.cs

```diff
@@ -5,25 +5,48 @@
 namespace System.Buffers
 {
     /// <summary>
-    /// Represents a <typeparam name="T"/> sink
+    /// Represents a <typeparam name="T"/> output sink into which data can be written.
```

Comment:
Should I add, `synchronously`?

Reply:
T is not a sink or output. It's the coding unity of the data being written.

Reply:
`Represents an output sink into which <typeparam name="T"/> data can be written`?

Reply:
But this doc comment is for type parameter T, not for the whole type IBufferWriter<T>, isn't it?

Reply:
No, this doc comment is for the whole type: `IBufferWriter<T>`

For example see:
https://docs.microsoft.com/en-us/dotnet/api/system.buffers.ibufferwriter-1?view=netcore-2.2
![image](https://user-images.githubusercontent.com/6527137/53379672-630bd500-391f-11e9-8b42-2b5ad48949d1.png)

Reply:
Ah, sorry. I missed that. Looks good then.